layout_catalog/src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,109 @@
+/* #region page__catalog */
+.page__catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  box-sizing: border-box;
```

Comment:
move it to * selector

Reply:
created * inside the cards.scss (I hope it`s correct)